fisher.fish

```diff
@@ -37,15 +38,20 @@ function fisher -a cmd -d "fish package manager"
         end
     end
 
+    # 2019-10-22: temp code, migrates fishfile from old path back to $fish_config
+    if test -e "$fisher_path/fishfile"; and test ! -e "$fish_config/fishfile"
+        command mv -f "$fisher_path/fishfile" "$fish_config/fishfile"
+    end
```

Comment:
Let me know if this works for you for fishfile migration @jorgebucaran.

Reply:
Seems alright.